test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/MaxRequestBufferSizeTests.cs

```diff
@@ -245,11 +245,6 @@ public static IEnumerable<object[]> LargeUploadData
         private static Socket CreateSocket(int port)
         {
             var socket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
-
-            // Timeouts large enough to prevent false positives, but small enough to fail quickly.
-            socket.SendTimeout = 10 * 1000;
```

Comment:
Why was this changed?  I think there might be small advantages to set the timeouts on the socket, otherwise I think the manage task might timeout but the native socket will stay around longer.